packages/craco/lib/features/webpack/merge-webpack-config.js

```diff
@@ -111,5 +113,5 @@ function mergeWebpackConfig(cracoConfig, webpackConfig, context) {
 }
 
 module.exports = {
-    mergeWebpackConfig
+    mergeWebpackConfig,
```

Comment:
```suggestion
    mergeWebpackConfig
```
Remove unnecessary autoformat.